hexa/pipelines/admin.py

```diff
@@ -19,15 +20,36 @@ def restore_pipelines(modeladmin, request, queryset):
         obj.restore()
 
 
+class DeletedPipelineFilter(admin.SimpleListFilter):
```

Comment:
Could you please take some inspiration from https://github.com/san4ezy/django_softdelete/blob/master/django_softdelete/admin.py & https://github.com/san4ezy/django_softdelete/blob/master/django_softdelete/filters.py to not have to write this piece of code for every SoftDeleteModel ?